_data/banking.yml

```diff
@@ -351,6 +351,13 @@ websites:
       software: Yes
       doc: https://www.handelsbanken.se/shb/INeT/ICentSv.nsf/Default/qAE99695C4C1D033DC12579120036EB43?Opendocument
 
+    - name: Holvi
+      url: https://www.holvi.com/
```

Comment:
I don't think the redirect is an issue, especially since this listing will probably always be live, even if they stop using the "about" subdomain in the future.